modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/display/context/RoleDisplayContext.java

```diff
@@ -189,9 +192,25 @@ public List<NavigationItem> getRoleAssignmentsNavigationItems(
 		String tabs2 = ParamUtil.getString(
 			_httpServletRequest, "tabs2", "users");
 
+		Map<String, String[]> parameterMap = portletURL.getParameterMap();
+
+		long roleId = GetterUtil.getLong(parameterMap.get("roleId")[0]);
```

Comment:
What about if in the request there is no the roleId parameter? Does the [0] raise an error?